### DISCUSSION TITLE
Can I retrieve the style part of a template from API? If not is there any way to add my CSS and retrieve it from API?

### DISCUSSION THREAD

**Discussion by shokufe** [mixcore/mix.core#336] [Q&A] [answered]
Hi, first thanks for your powerful CMS. I have a question,
Can I retrieve the style part of a template from API? If not is there any way to add my CSS and retrieve it from API? 
The style existed in here(for each page): configuration->Edit Page Template-> style

**Comment by Smilefounder** [comment, marked as answer]
Hi @shokufe Yes, can. It is part from Model.View.Styles . You can ref from the sample code below:

```razor
@model Mix.Cms.Lib.ViewModels.MixModules.ReadMvcViewModel

@await Html.PartialAsync(Model.TemplatePath, Model)
@Html.Raw(Model.View.Scripts) 
@Html.Raw(Model.View.Styles) 
```

Example: https://dev.mixcore.org/api/v1/rest/en-us/mix-page/mvc/7

![image](https://user-images.githubusercontent.com/3785721/107740755-70a20b80-6d3e-11eb-93b0-f45e3bcd6a0d.png)

**Comment by shokufe** [comment]
hi Huy, thanks for your answer. but when I added a new style it's still empty. I put an image here
![mixPrb2](https://user-images.githubusercontent.com/2022244/107748818-ef388200-6d19-11eb-82ca-e007d464b82b.JPG)
![mixProb1](https://user-images.githubusercontent.com/2022244/107748819-efd11880-6d19-11eb-9ec5-3316f949581f.JPG)

**Reply by nhathoang989** [reply]
Hi @shokufe,
I think there's an issue with cache system
Please turn off the cache and try again,
I'll fix it later
![image](https://user-images.githubusercontent.com/5327699/107749544-5ae81280-6d4d-11eb-875c-6262d5beaced.png)

**Reply by Smilefounder** [reply]
I can not reproduce the issue from my side, see below:
(you might need to install https://chrome.google.com/webstore/detail/json-viewer/gbmdgpbipfallnflgajpaliibnhdgobh for easier to see the return JSON)

![image](https://user-images.githubusercontent.com/3785721/107749497-4a379c80-6d4d-11eb-93cd-4bc05addf848.png)

![image](https://user-images.githubusercontent.com/3785721/107749555-5d4a6c80-6d4d-11eb-81bd-d00091820bc5.png)
